deps/rabbit/src/rabbit_fifo_maps.erl

```diff
@@ -29,13 +30,18 @@ keys(Map, Vsn) ->
     AccIn :: Init | AccOut,
     Map :: #{Key => Value}.
 fold(Fun, Init, Map, Vsn)  ->
-    Iterable = case is_deterministic(Vsn) of
-                   true ->
-                       maps:iterator(Map, ordered);
-                   false ->
-                       Map
-               end,
-    maps:fold(Fun, Init, Iterable).
+    maps:fold(Fun, Init, iterator(Map, Vsn)).
```

Comment:
Note that `maps:fold/3` creates an iterator with `maps:iterator/1` by default, so this is equivalent to the old code which passed a map directly: https://github.com/erlang/otp/blob/d4c78e0414664564ae29d4d6a09bbaf7d2c2110b/lib/stdlib/src/maps.erl#L881-L882